lib/winner.rb

```diff
@@ -0,0 +1,19 @@
+class Winner
+  attr_reader :choices
+  # rubocop:todo Style/ClassVars
+  # rubocop:todo Naming/VariableName
+  @@WINNING_POS = [[1, 2, 3], [4, 5, 6], [7, 8, 9], [1, 4, 7], [2, 5, 8], [3, 6, 9], [1, 5, 9], [3, 5, 7]]
```

Comment:
Declare it as a **CONSTANT**

Read this [article](http://rubylearning.com/satishtalim/ruby_constants.html) to know create RUBY CONSTANT